pkg/daemon/daemon.go

```diff
@@ -183,18 +183,30 @@ func (d *daemon) AddPeriodicUpdate() {
 					continue
 				}
 			} else {
-				allocatedGuid, err = d.guidPool.GenerateGUID("")
+				guidAddr, err = d.guidPool.GenerateGUID()
 				if err != nil {
 					glog.Error(err)
 					continue
 				}
+				allocatedGuid = guidAddr.String()
+				if guidErr := d.guidPool.AllocateGUID(string(pod.UID)+networkName, allocatedGuid); guidErr != nil {
```

Comment:
Pass the podid and network each one as arg

Reply:
added